data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg

```diff
@@ -15,7 +15,7 @@
 
     [story]
         [part]
-            story= _ "After reaching the dwarvish keep and tending to their wounded, the humans led by Tallin and the dwarves led by Lord Hamel hurried deeper into the caverns for a hasty council."
+            story= _ "After reaching the dwarvish keep and tending to their wounded the humans, led by Tallin, and the dwarves led by Lord Hamel hurried deeper into the caverns for a hasty council."
```

Comment:
Removing that comma seems to be a grammatical error, the result doesn't scan at all to me.

The commas around "led by Tallin" don't add anything IMO, so in all respects I prefer the original string.

Reply:
The first comma needs to come back, however the second makes a comma appositive and needs to be added about Lord Hamnel. I have no idea how I missed that.